bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -43,7 +41,7 @@ def create(self):
             stdout=self.stdout,
             stderr=self.stderr,
         )
-        logging.info(f"Started OpenSearch with PID {self.process.pid}")
+        logging.info(f"Started OpenSearch with parentPID {self.process.pid}")
```

Comment:
@setiah Does this clear the confusion with PID? 
nit: @owaiskazi19 Either say `PPID` or `parent PID`